### ISSUE TITLE
About submission on the kitti dataset

### ISSUE THREAD

**Issue by DrugRui** [JiaRenChang/PSMNet#158]
Hello, I would like to ask whether you used all the training data of kitti12 and kitti15 or only the training data of kitti15 when submitting the results at kitti15?Looking forward to your reply

**Comment by JiaRenChang** [comment]
@DrugRui 
Yes, only KITTI 2015 dataset for training the model of KITTI 2015 submission.

**Comment by DrugRui** [comment]
> @DrugRui
> Yes, only KITTI 2015 dataset for training the model of KITTI 2015 submission.

Thank you!

**Comment by DrugRui** [comment]
Hello, may I ask if the results uploaded to kitti need to be turned into RGB？

**Comment by Chenlei6** [comment]
I used your submission.py to run the KITTI2012 dataset, but when I submit it to the KITTI benchmark, the website always reminds me something wrong with the data format. Have you had a similar problem?

**Comment by GANWANSHUI** [comment]
> I used your submission.py to run the KITTI2012 dataset, but when I submit it to the KITTI benchmark, the website always reminds me something wrong with the data format. Have you had a similar problem?

Hi, have you solve the problem?

**Comment by Chenlei6** [comment]
> ar problem

No, can you give some advice?

**Comment by Chenlei6** [comment]
@GANWANSHUI 
No, can you give some advice?

**Comment by DrugRui** [comment]
On kitti submit results, I put all the pictures are the result of the compression to disp_0.zip. The image name has been checked and there is no problem. The final submission was successful, but the email received was as follows:
"Thank you for participating in our evaluation!"
Find 0/200 images in the dis_0 folder.
Find 0/200 images in the dis_1 folder.
Find 0/200 images in the stream folder.
Without enough result images to find any evaluation, stop the evaluation.
An error occurred while processing the result.
Make sure the data in your zip file is properly formatted!"
Could you give me some help? Looking forward to your reply.@JiaRenChang

**Comment by Chenlei6** [comment]
Do you mean that you have successfully submitted the results but received an email notification that the submission failed? Can you see the metrics of submission results on the website？





------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"DrugRui"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月23日(星期六) 上午10:47
收件人:&nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&gt;;
抄送:&nbsp;"谌"<954580312@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158)




On kitti submit results, I put all the pictures are the result of the compression to disp_0.zip. The image name has been checked and there is no problem. The final submission was successful, but the email received was as follows:
 "Thank you for participating in our evaluation!"
 Find 0/200 images in the dis_0 folder.
 Find 0/200 images in the dis_1 folder.
 Find 0/200 images in the stream folder.
 Without enough result images to find any evaluation, stop the evaluation.
 An error occurred while processing the result.
 Make sure the data in your zip file is properly formatted!"
 Could you give me some help? Looking forward to your reply.@JiaRenChang

**Comment by DrugRui** [comment]
Thank you for your reply. The problem has been solved, and I can see the indicators after resubmitting.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Chenlei6"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月26日(星期二) 上午10:53
收件人:&nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&gt;;
抄送:&nbsp;"Drug."<945425345@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158)



Do you mean that you have successfully submitted the results but received an email notification that the submission failed? Can you see the metrics of submission results on the website？ 
 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:&amp;nbsp;"DrugRui"<notifications@github.com&amp;gt;; 
 发送时间:&amp;nbsp;2019年11月23日(星期六) 上午10:47 
 收件人:&amp;nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&amp;gt;; 
 抄送:&amp;nbsp;"谌"<954580312@qq.com&amp;gt;;"Comment"<comment@noreply.github.com&amp;gt;; 
 主题:&amp;nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158) 
 
 
 
 
 On kitti submit results, I put all the pictures are the result of the compression to disp_0.zip. The image name has been checked and there is no problem. The final submission was successful, but the email received was as follows: 
  "Thank you for participating in our evaluation!" 
  Find 0/200 images in the dis_0 folder. 
  Find 0/200 images in the dis_1 folder. 
  Find 0/200 images in the stream folder. 
  Without enough result images to find any evaluation, stop the evaluation. 
  An error occurred while processing the result. 
  Make sure the data in your zip file is properly formatted!" 
  Could you give me some help? Looking forward to your reply.@JiaRenChang

**Comment by Chenlei6** [comment]
Do you attempt to submit the results of KITTI2012? Did it succeed?

**Comment by DrugRui** [comment]
No, I only tried it on kt15.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Chenlei6"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月26日(星期二) 中午11:28
收件人:&nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&gt;;
抄送:&nbsp;"Drug."<945425345@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158)



Do you attempt to submit the results of KITTI2012? Did it succeed?

**Comment by Chenlei6** [comment]
Can you tell me how you solve the problem? I hope to learn from your solution to deal with the problem I encountered!

**Comment by DrugRui** [comment]
I submitted it on a MAC and it didn't work, and then I tried Windows and it worked. First, put all the pictures in disp_0 folder, and then compress the folder to submit.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Chenlei6"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月27日(星期三) 中午12:16
收件人:&nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&gt;;
抄送:&nbsp;"Drug."<945425345@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158)



Can you tell me how you solve the problem? I hope to learn from your solution to deal with the problem I encountered!

**Comment by Chenlei6** [comment]
Ok! Thanks for your help!

**Comment by huangcimu** [comment]
hello,DrugRui,I want to ask you what the resolution of the image you submitted to KITTI2012 for evaluation is, as if each parallax image has a different resolution

**Comment by DrugRui** [comment]
Sorry, I only submitted on kt15, the resolution is the same as the original, it is 1242 × 375




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"huangcimu"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月27日(星期五) 下午2:22
收件人:&nbsp;"JiaRenChang/PSMNet"<PSMNet@noreply.github.com&gt;;
抄送:&nbsp;"Drug."<945425345@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [JiaRenChang/PSMNet] About submission on the kitti dataset (#158)




hello,DrugRui,I want to ask you what the resolution of the image you submitted to KITTI2012 for evaluation is, as if each parallax image has a different resolution

**Comment by kriswangjie** [comment]
Hello,dear author, I have a problem in submitting KITTI dataset's disparity result file. Could you tell me the organization of the submission file? When I submit the result file of KITTI dataset, I compress the all disparity result image into a zip file. After uploading, I get nothing.

**Comment by SamChuah** [comment]
@kriswangjie did you put all your results in a directory named "disp_0"? You should do that before zipping the file. 

Results(PNGs) --> Directory(disp_0) --> zipping --> zip file(_**your_results**_.zip)
